app.js

```diff
@@ -14,6 +15,21 @@ promBundle.promClient.collectDefaultMetrics();
 // see https://github.com/jochen-schweizer/express-prom-bundle#sample-uusage
 app.use(promBundle.middleware);
 
+app.use(helmet.noCache());
+app.use(helmet.contentSecurityPolicy({
+  directives: {
+    defaultSrc: ['\'self\''],
+    imgSrc: ['\'self\'', 'data:'],
+    styleSrc: ['\'unsafe-inline\''],
+  },
+}));
+app.use(helmet.frameguard({ action: 'deny' }));
+app.use(helmet.hidePoweredBy());
+app.use(helmet.ieNoOpen());
+app.use(helmet.noSniff());
+app.use(helmet.hsts());
```

Comment:
I'm assuming this is adding the HSTS header?

Reply:
It is. As a bunch of other apps are already serving it.

Reply:
As discussed, this module will continue to be run but will be configured to not include subdomains.